nibabel/freesurfer/tests/test_mghformat.py

```diff
@@ -23,7 +23,8 @@
 from ...wrapstruct import WrapStructError
 from ... import imageglobals
 
-from nose.tools import assert_true, assert_false
+
+import pytest
 
 from numpy.testing import (assert_equal, assert_array_equal,
                            assert_array_almost_equal, assert_almost_equal,
```

Comment:
Here, we need to replace `assert_equal`, `assert_raises` and `assert_not_equal`, and again update `...testing`.

Reply:
Ok. I didn't touch the other asserts  because its numpy.testing - Should I convert all of those accordingly and remove nupy.testing altogether?

Reply:
We should keep `assert_array_equal`, `assert_array_almost_equal` and `assert_almost_equal`, since these are useful for arrays and floating point comparisons, which don't have `unittest`/`pytest` equivalents.